javaprocessor/Dockerfile

```diff
@@ -16,10 +16,8 @@
 FROM ubuntu:xenial
 
 RUN \
- echo "deb http://repo.sawtooth.me/ubuntu/ci xenial universe" >> /etc/apt/sources.list \
```

Comment:
You removed this.
It is needed for the protobuf package.
Is this not needed any longer?

Reply:
not required now, since I'm using the SDK/protos directly from maven repo.